.github/workflows/test-environment.yml

```diff
@@ -442,6 +429,16 @@ jobs:
         run: |
           poetry run pytest -m "sanity" --alluredir=./allure/results/ --clean-alluredir --maxfail=4
 
+      - name: Run UI Sanity checks (Kibana)
+        uses: ./.github/actions/kibana-ftr
+        if: ${{ success() && inputs.run-ui-sanity-tests == true }}
+        with:
+          test_kibana_url: ${{ env.TEST_KIBANA_URL }}
+          test_es_url: ${{ env.TEST_ES_URL }}
+          es_version: ${{ env.STACK_VERSION }}
+          ref_commit_sha: ${{ inputs.ref_commit_sha }}
```

Comment:
Version control for Kibana UI tests has become relevant because the UI controls implementation can differ between versions. Currently, I assume that the user executing the workflow will select the correct Kibana version and branch, for example, `8.15.0-SNAPSHOT` and `main`, or in the future `8.15.0` and `v8.15.0` in Kibana.
I am considering whether we should implement logic to prevent execution if versions do not match correctly. In the current create environment workflow, we allow installing different versions, so we need to decide if we should add additional logic to enforce version consistency, or if we will simply allow the UI tests to fail if an incorrect branch/tag is provided.